testcases/baitap2.robot

```diff
@@ -0,0 +1,40 @@
+*** Settings ***
+Library     SeleniumLibrary
+Library     OperatingSystem
+Library     Collections
+
+Suite Setup       Run Keywords      Navigate To Homepage
+Suite Teardown    Exit Selenium
+
+*** Variables ***
+${SiteURL}    http://live.techpanda.org/index.php/customer/account/login/
+${Browser}    Chrome
+${LOCATOR1}    //h2[@class='product-name']//a[normalize-space()='Sony Xperia']
+${LOCATOR2}    //p[@class='product-name']//a[normalize-space()='Sony Xperia']
+
+*** Keywords ***
+Navigate To Homepage
+    Open Browser               ${SiteURL}    ${BROWSER}
+    Maximize Browser Window
+
+Compare text
```

Comment:
@trungdb-2300 Compare text này nên là compare text của item **trước khi** click add to cart và text của item trong cart popup, không phải so sánh 2 text đều trong cùng cart